app/[locale]/layout.tsx

```diff
@@ -68,20 +68,15 @@ export default async function LocaleLayout({
         <NextIntlClientProvider messages={messages}>
           <AuthProvider>
             <CsrfProvider>
-
-              <Header />
-              <SubHeader />
-
-              <div id="App">
-                <ApolloWrapper>
-
+              <ApolloWrapper>
```

Comment:
ApolloWrapper should wrap the Header so we can use graphql requests in header as well